packages/universal-xml-plugin/lib/plugin.js

```diff
@@ -1,20 +1,13 @@
+// @ts-check
 /* eslint-disable no-case-declarations */
 
-import BasePlugin from 'appium/plugin';
+import {BasePlugin} from 'appium/plugin';
 import {errors} from 'appium/driver';
 import {transformSourceXml} from './source';
 import {transformQuery} from './xpath';
 import log from './logger';
 
 export default class UniversalXMLPlugin extends BasePlugin {
-  commands = [
```

Comment:
how this assignment should work now?

Reply:
this code was unused afaict.

Reply:
yeah i think it came from an old version of our base plugin logic which required registration of overrides.